frontend/assets/card-draft.inkscape.svg

```diff
@@ -0,0 +1,226 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Do we want to keep that file?

Reply:
Hmm I can remove again, no problem. It was just a draft for my head :D

Reply:
You do not have to. Might be useful for future developers to see what it looks like. I just wondered if there is a specific reason for that file to be there that I failed to see.